test/test-entity.js

```diff
@@ -0,0 +1,53 @@
+/*global describe, beforeEach, it*/
+'use strict';
+
+var path = require('path');
+var assert = require('yeoman-assert');
+var helpers = require('yeoman-test');
+var os = require('os');
+var fse = require('fs-extra');
+
+describe('JHipster generator entity', function () {
+    describe('default configuration', function () {
+        before(function (done) {
+            helpers.run(require.resolve('../entity'))
+                .inTmpDir(function (dir) {
+                  fse.copySync(path.join(__dirname, '../test/templates/default'), dir)
+                })
+                .withArguments(['foo'])
+                .withPrompts({
+                    "fieldAdd": false,
```

Comment:
You don't need quotes around keys : `"fieldAdd": false,` => , `fieldAdd: false,` unless your key has a character that would break your linter if you have one, like a `_`.
Also, I suggest choosing between double quotes and single quotes, and stick to it.